Changelog.md

```diff
@@ -1,3 +1,9 @@
+### 0.5.15 (2019-12-17)
+
+Bugfixes:
+ * Yul Optimizer: Fix redundant load optimization crossing user-defined functions that contain for-loops.
```

Comment:
```suggestion
 * Yul Optimizer: Fix incorrect load optimization crossing user-defined functions that contain for-loops.
```

Reply:
incorrect redundant load optimization?

Reply:
I think `redundant` is redundant if we start with the adjective `incorrect` (because at least to me, incorrect is a stronger adjective in this context) :-)